huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -707,6 +612,11 @@ func resourceRdsInstanceRead(ctx context.Context, d *schema.ResourceData, meta i
 		return diag.Errorf("error saving nodes to RDS instance (%s): %s", instanceID, err)
 	}
 
+	return setRdsInstanceParameters(ctx, d, client, instanceID)
+}
+
+func setRdsInstanceParameters(ctx context.Context, d *schema.ResourceData, client *golangsdk.ServiceClient,
+	instanceID string) diag.Diagnostics {
 	// Set Parameters
 	configs, err := instances.GetConfigurations(client, instanceID).Extract()
 	if err != nil {
```

Comment:
we can return a nil in this case and the else case is unnecessary.

Reply:
done